examples/2_Iterative/02_tikhonov_block_framework.ipynb

```diff
@@ -106,12 +105,11 @@
     "from cil.plugins.astra import ProjectionOperator\n",
     "\n",
     "# For shepp-logan test image in CIL tomophantom plugin\n",
-    "import cil.plugins.TomoPhantom as TP\n",
+    "from cil.plugins import TomoPhantom\n",
```

Comment:
I'd prefer importing this as e.g. cilTomoPhantom, or CTP or something, to clarify when using it for example in line 161 that it is not TomoPhantom itself that is being used but the CIL TomoPhantom plugin

Reply:
When Gemma and I looked at this we changed it from TP to TomoPhantom because when we first looked at the code further down we both had no idea what TP was until we looked at the import.

So I think cilTomoPhantom would be the better option, rather than CTP so it is readable.

Reply:
I will change it!